poc/rsabssa.py

```diff
@@ -57,7 +58,10 @@ def RSAVP1(public_key: RsaKey, r: int) -> int:
     return pow(r, e, n)
 
 
-def random_integer_uniform(min: int, max: int) -> int:
+def random_integer_uniform(m: int, n: int) -> int:
+    # Implement rejection sampling.
+    # This is for reference only; most cryptographic libraries include
+    # functions to generate random integers from a uniform distribution.
     range = max - min
     rangeBits = size(range - 1)
```

Comment:
You may want to indicate in the comment which of the intervals `[m, n)` and `[m, n]` this function samples from.

Reply:
Added, thank you!